gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.2.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
```

Comment:
i'm not really sure, since i was not going with loom 1.6 i just went with the one loom said was the minimum requirement, sometimes newer versions of gradle can mess with older versions of loom ex. the deprecation of self-resolving dependency.